### DISCUSSION TITLE
Verification email not being sent for api registration

### DISCUSSION THREAD

**Discussion by murraygunn** [laravel/framework#49240] [Q&A]
I've created a project with breeze and react, which works as expected. I've also created a react native app that works as expected apart from one thing. When a new user registers, they are created and the response is returned correctly. From telescope, it also appears that a mailable is created as part of the verification process, however it's never received. I've reproduced this with a minimal project as follows.

1. create project (composer create-project laravel/laravel Minimal)
2. configure .env file with email server and account
3. install breeze (composer require laravel/breeze --dev)
4. add react (php artisan breeze:install react) (I assume the issue is the same for vue and blade)
5. complete migrations (php artisan migrate)
6. configure the User model for verification
`use Illuminate\Contracts\Auth\MustVerifyEmail;`
`class User extends Authenticatable implements MustVerifyEmail`
8. add route in routes/api.php pointing api registration to breeze's registration method (or reproduce in separate controller and point to that)
`Route::post('register', [RegisteredUserController::class, 'store']);`
9. start server (php artisan serve)
10. register from web, then check to see email received
11. send same request to '/api/register', then check to see user created and telescope shows the same data as for web registration (response 302 with user details and a 'Illuminate\Auth\Notifications\VerifyEmail' mailable) but no email received.

According to the docs at https://laravel.com/docs/10.x/verification#model-preparation, `event(new Registered($user));` should be enough to trigger the email properly, but it's not working for api requests. Nor is `$request->user()->sendEmailVerificationNotification();` which should trigger the same thing manually.

Since the event, listener and mail are all in Illuminate, I don't know how to troubleshoot them. I also tried using `Mail::assertSent()` from https://laravel.com/docs/10.x/mail#testing-mailable-sending, but I get an error saying that method doesn't exist. I've tried posting this in StackOverflow, but no one has responded, so I'm hoping there are more laravel-focused people here.

Am I missing something in setting up my api (which works for everything else including sending an email I've set up myself)? How can I troubleshoot this further? Or is it a bug?

**Comment by Dev-Kings** [comment]
Maybe this can help 👉 https://stackoverflow.com/questions/69992374/laravel-8-sendverificationemail-event-seemingly-not-firing
